src/main/java/frc/robot/commands/CompressorTurnOffWhileShootingOrClimbingCommand.java

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) FIRST and other WPILib contributors.
```

Comment:
Consider creating two custom triggers: https://docs.wpilib.org/en/2020/docs/software/commandbased/binding-commands-to-triggers.html#creating-your-own-custom-trigger
https://docs.wpilib.org/en/2020/docs/software/commandbased/convenience-features.html

`Trigger isClimbing= new Trigger(Robot.COMPRESSOR_SUBSYSTEM::isClimbing);`
`Trigger isShooting= new Trigger(Robot.COMPRESSOR_SUBSYSTEM::isShooting);`
then you can chain them together
```
isClimbing.and(isShooting)
.whenActive(Robot.COMPRESSOR_SUBSYSTEM::start)
.whenInactive(Robot.COMPRESSOR_SUBSYSTEM::stop);
```